e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/unauthenticatedswaggeraccess/UnauthenticatedSwaggerAccessRest.java

```diff
@@ -0,0 +1,27 @@
+package com.foo.rest.examples.spring.unauthenticatedswaggeraccess;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/api")
+public class UnauthenticatedSwaggerAccessRest {
+
+    @GetMapping(value = "/endpoint1")
+    public ResponseEntity getResourceEnd1(@PathVariable("x") String x) {
```

Comment:
same comments as other class

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.